standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -572,6 +572,9 @@ public static ConfVars getMetaConf(String name) {
             "Set this to true on one instance of the Thrift metastore service as part of turning\n" +
             "on Hive transactions. For a complete list of parameters required for turning on\n" +
             "transactions, see hive.txn.manager."),
+    COMPACTOR_INITIATOR_TABLE_CACHE_ON("metastore.compactor.initiator.table.cache.on",
```

Comment:
Could you please change the config value to:
````
COMPACTOR_INITIATOR_TABLE_CACHE_ON(
“metastore.compactor.initiator.tablecache.on”, “hive.compactor.initiator.tablecache.on”, true ..
````

Reply:
How confident are we in this feature?
Shouldn't we default to false in the beginning?

Reply:
I think it should be "on" by default, as it's a known limitation. Currently, it's an isolated, simple cache with soft-ref, so shouldn't be risky.

Reply:
your call 😄